### PR TITLE
Introduce embedded-compositor profile environment file

### PR DESCRIPTION
**Pull request by ChHapp**

### CODE REVIEW

File: recipes-core/embedded-compositor/files/embedded-compositor-profile.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export XDG_RUNTIME_DIR=/run/user/0
+export XDG_CONFIG_HOME=/run
```
Comment (EugenWiens) [inline recipes-core/embedded-compositor/files/embedded-compositor-profile.sh:2]: shold we add the rotation environment, here with 0° ?

https://github.com/JUMO-GmbH-Co-KG/embedded-compositor#screen-orientation

`SCREEN_ORIENTATION`

**Review by EugenWiens — APPROVED** [review]
looks good for me

File: recipes-core/embedded-compositor/files/embedded-compositor-profile.sh (hunk 1 above)
Comment (ChHapp) [reply recipes-core/embedded-compositor/files/embedded-compositor-profile.sh:2]: I will put this in the next PR

**Comment by ChHapp** [comment]
This approch seems not to work as intended, therefore I close it now

**Pull request closed without merging**